src/assets/scss/foundations.scss

```diff
@@ -208,6 +212,14 @@ table {
 		background-color: var(--lightest-background-color);
 		padding: 0.25rem 0.5rem;
 	}
+
+	@media screen and (max-width: 768px) {
+		display: block;
+		width: fit-content;
+		max-width: 100%;
+		margin-inline: auto;
+		overflow-x: scroll;
+	}
```

Comment:
```suggestion
        tbody tr > td:first-child {
            border-left: none;
        }

        tbody tr > td:last-child {
            border-right: none;
        }
	}
```
There is a minor issue with the table. In the table body, there are extra borders on the left and right sides. By applying the above CSS, we can remove these unnecessary borders.


| before  | After |
--------|------
| ![image](https://github.com/user-attachments/assets/892dc851-0e2a-4ced-816b-988ba1085429)  | ![image](https://github.com/user-attachments/assets/5156a2c8-224a-486c-bf27-4fe425ebf5c8) |
| ![image](https://github.com/user-attachments/assets/3c4bb244-cc40-4da7-911c-6691a76d55ac) | ![image](https://github.com/user-attachments/assets/56c555b1-21f6-4fca-8a2d-4fdd62f1c2f4) |